src/style.css

```diff
@@ -2,0 +1,40 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.rating-star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.rating-star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .rating-star:nth-child(-n + 1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .rating-star:nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 .rating-star:nth-child(-n + 3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 .rating-star:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 .rating-star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
These selectors are well-organized, but to improve maintainability and reduce repetition, consider using a preprocessor like SASS. This would allow you to use loops for generating these classes, which would make the code shorter and easier to maintain.